Makefile

```diff
@@ -0,0 +1,90 @@
+# ============================================================================ #
```

Comment:
Did you mean to commit this in this PR?

Reply:
Yes at the time.
Until this morning we had decided on using a top level `Makefile`.

Demo time was contradicted that agreement, so I can remove it until we settle that.